docs/hive-integration/kube-api-getting-started.md

```diff
@@ -0,0 +1,493 @@
+time="2021-06-28T21:42:20Z" level=info msg="InfraEnv Reconcile ended" func="github.com/openshift/assisted-service/internal/controller/controllers.(*InfraEnvReconciler).Reconcile.func1" file="/go/src/github.com/openshift/origin/internal/controller/controllers/infraenv_controller.go:78" go-id=654 infra_env=myinfraenv infra_env_namespace=demo-worker4 request_id=5ad649fd-bf49-4b41-aa33-c35a601c1555
+time="2021-06-28T21:42:20Z" level=info msg="InfraEnv Reconcile started" func="github.com/openshift/assisted-service/internal/controller/controllers.(*InfraEnvReconciler).Reconcile" file="/go/src/github.com/openshift/origin/internal/controller/controllers/infraenv_controller.go:81" go-id=654 infra_env=myinfraenv infra_env_namespace=demo-worker4 request_id=0d9c0da4-beab-4a41-85a3-4444728dd6f0
+time="2021-06-28T21:42:20Z" level=info msg="the amount of nmStateConfigs included in the image is: 1" func="github.com/openshift/assisted-service/internal/controller/controllers.(*InfraEnvReconciler).ensureISO" file="/go/src/github.com/openshift/origin/internal/controller/controllers/infraenv_controller.go:288" go-id=654 infra_env=myinfraenv infra_env_namespace=demo-worker4 request_id=0d9c0da4-beab-4a41-85a3-4444728dd6f0
+```
+
+###### Inspect Assisted Service Cluster Events
+Get the full event list here:
+```bash
+curl -s -k $(kubectl -n demo-worker4 get agentclusterinstalls.extensions.hive.openshift.io test-agent-cluster-install -o=jsonpath="{.status.debugInfo.eventsURL}")  | jq "."
+```
+
+At this point, you should expect to see events such as:
+```json
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:01:31.938Z",
+    "message": "Registered cluster \"test-cluster\"",
+    "severity": "info"
+  },
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:01:31.941Z",
+    "message": "Successfully registered cluster test-cluster with id 02a89bb9-6141-4d14-a82e-42f254217502",
+    "severity": "info"
+  },
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:42:20.768Z",
+    "message": "Generated image (Image type is \"minimal-iso\", SSH public key is not set)",
+    "severity": "info"
+  },
+```
+
+
+### Utilize BareMetalHost
```

Comment:
Might be a good place to emphasize that this part is optional, and the user can otherwise boot the image manually.

Reply:
In a separate section now